lib/posix/Kconfig

```diff
@@ -42,6 +42,28 @@ config SEM_VALUE_MAX
 
 endif # PTHREAD_IPC
 
+config POSIX_PIPE_IPC
```

Comment:
Please drop this "IPC" suffix. It's presence in CONFIG_PTHREAD_IPC is historical artifact (which should be fixed). Just call it "POSIX pipe()".